### DISCUSSION TITLE
How to enable AutoPlay?

### DISCUSSION THREAD

**Discussion by IncredibleNinja** [owncast/owncast#1296] [Q&A]
is there a way to enable auto play with and/or without audio for the the player? both for the embed code and the direct page?

**Comment by jeyemwey** [comment]
Most browsers block media autoplay [[e.g. Firefox]](https://support.mozilla.org/en-US/kb/block-autoplay), or at least block autoplay if the video is unmuted. To provide a consistent interface, we have decided to not use autoplay.

You can attempt to change that setting in [webroot/js/components/player.js, line 16](https://github.com/owncast/owncast/blob/develop/webroot/js/components/player.js#L16). However, it might not work for various reasons.

The player component is used both by the Owncast page and by the embed page.

**Reply by gabek** [reply]
It's an old blog post, but there are some suggestions here that might help you in configuring vjs: https://videojs.com/blog/autoplay-best-practices-with-video-js/
